esbuild.js

```diff
@@ -0,0 +1,13 @@
+const { getOptions } = require('./scripts/build/utils/options.js');
```

Comment:
I just took a look back at this and ended up deciding to move this to `scripts/esbuild/build.ts` instead of having it live in the root as a `.js` file - that way we can use ESM imports and have some type-checking and whatnot, and we can just run the built output in `scripts/build/esbuild/build.js`